native/src/components/List.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react'
 import { ReactNode } from 'react'
-import { isEmpty } from 'lodash/lang'
+import { isEmpty } from 'lodash'
```

Comment:
regarding lodash. In react-native there is no tree shaking. Which basically it makes no difference wether you import from `lodash` or `lodash/lang` . Actually the bungle size gets even smaller when importing only from lodash, which is probably because the optimizer can do its job better or sth. https://github.com/facebook/metro/issues/227#issuecomment-583358386